manifests/0000_90_kube-controller-manager-operator_06_cluster-policy-controller-rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: cluster-policy-controller-leader-lock-cm
+  namespace: openshift-kube-controller-manager
+subjects:
+- kind: User
+  name: system:serviceaccount:openshift-controller-manager:openshift-controller-manager-sa
```

Comment:
no.  This belongs in the openshift-controller-manager-operator eventually.  Wait until we're done here to open that.

Reply:
ok

Reply:
done